components/tools/OmeroPy/src/omero/install/jvmcfg.py

```diff
@@ -398,6 +398,12 @@ def adjust_settings(config, template_xml,
             ("pixeldata", pixeldata), ("repository", repository))
 
     for name, StrategyType in loop:
+        if name not in options:
+            raise Exception(
+                "Cannot find %s option. Make sure templates.xml was "
+                "not copied from a 5.0.2 or older server" % name)
```

Comment:
Thinking about this again. I agree, if we make further changes to `templates.xml` in the future, we will need to make sure this is in-sync with the current server anyways.

Actually a related thought: should we `chmod 444 templates.xml` since this file should only be used in read-only mode? /cc @joshmoore